app/src/gui/components/notebook/index.tsx

```diff
@@ -205,18 +223,55 @@ export default function NotebookComponent(props: NotebookComponentProps) {
                 textColor="inherit"
                 variant="scrollable"
                 scrollButtons="auto"
+                sx={{
+                  '& .MuiTabs-indicator': {
+                    ...bssTabStyling.indicator,
+                  },
+                }}
               >
-                <Tab label="Records" {...a11yProps(0, NOTEBOOK_NAME)} />
-                <Tab label="Details" {...a11yProps(1, NOTEBOOK_NAME)} />
-                <Tab label="Settings" {...a11yProps(2, NOTEBOOK_NAME)} />
+                <Tab
+                  label={`${RECORD_LABEL}s`}
+                  {...a11yProps(0, NOTEBOOK_NAME)}
+                  sx={{...bssTabStyling.tabRoot}}
+                />
+                <Tab
+                  label="Details"
+                  {...a11yProps(1, NOTEBOOK_NAME)}
+                  sx={{...bssTabStyling.tabRoot}}
+                />
+                <Tab
+                  label="Settings"
+                  {...a11yProps(2, NOTEBOOK_NAME)}
+                  sx={{...bssTabStyling.tabRoot}}
+                />
               </Tabs>
             </AppBar>
           </Box>
+          {/* Display counts here */}
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+              backgroundColor: '#f5f5f5',
+              padding: '12px 16px',
+              borderRadius: '4px',
+              marginBottom: '16px',
+            }}
+          >
+            <Typography variant="body2" sx={{fontSize: '1.1rem'}}>
+              <strong>My {RECORD_LABEL}s:</strong> {myRecords}
```

Comment:
Use the form name from the UISpec here to get a name for the kind of entity we're listing.  In general we can only do this if there is only one kind of record in the list.  Otherwise it will have to say My Records.